docs/Contributing/pull-request.rst

```diff
@@ -0,0 +1,182 @@
+How to Submit a Pull Request
+****************************
+
+A Pull Request (PR), also commonly known as a Merge Request, is a mechanism to "pull" code from one repository
+into another one and merge the change into the source repository.
+
+Before we go much further, let's take a few minutes to describe what happened when you :doc:`dev-env` and to
+define a few terms that will help with futher instructions. Starting with a diagram of the setup of the three
+locations where code resides:
+
+.. image:: ../_static/images/github_repos.png
+    :align: center
+    :height: 300
+    :alt: Diagram of GitHub Repositories
+
+As part of the :doc:`../General/install` process, one of the programs installed is :file:`git`. This program
+is used to interact with the varying places that code can reside. The :file:`git` program has a relatively
+standard way of referring to things that are in the diagram. If you have not worked with :file:`git` much,
+you can read the `documentation <https://git-scm.com/docs>`_.
```

Comment:
This RTFM is a bit hard. I'd link to [the Pro Git book](https://git-scm.com/book/en/v2) and add "don't worry, you only need the first three chapters and they are quite short". Or something like that…

Reply:
fixed in commit